BookingsApi/BookingsApi.DAL/Migrations/20210729101118_UpdatePanelMemberUserRoleId.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System.Linq;
+using BookingsApi.DAL.Helper;
+using BookingsApi.Domain.RefData;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace BookingsApi.DAL.Migrations
+{
+    public partial class UpdatePanelMemberUserRoleId : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            var newUserRoleId = UserRoleForHearingRole.UserRoleId["Panel Member"];
+            migrationBuilder.UpdateData(
+                table: nameof(HearingRole),
+                keyColumn: "Id",
+                keyValue: 709,
+                column: "UserRoleId",
+                value: 7);
```

Comment:
should we change this to newUserRoleId or is this intentional?

Reply:
Its typo